src/pudl/metadata/resources/eiaaeo.py

```diff
@@ -0,0 +1,524 @@
+"""Table definitions for EIA AEO resources."""
+
+from typing import Any
+
+# 2024-04-24: workaround so we can *define* a bunch of schemas without
+# documenting them or expecting assets to exist
+_STAGING_RESOURCE_METADATA: dict[str, dict[str, Any]] = {
+    "core_eiaaeo__yearly_projection_electric_sector_generation_by_technology": {
+        "description": (
+            "Projected generation capacity & total generation in the electric "
+            "sector, broken out by technology."
+        ),
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
```

Comment:
nice

Reply:
hm i know we literally came up with `modeling_case_aeo` yesterday, but a) it should almost certainly have the full `_eiaaeo` (🧑‍🌾) suffix and maybe for a light attempt at more brevity it would be `model_case_eiaaeo`? 

also the full `_eiaaeo` (🧑‍🌾) suffix note i believe should be applied to many of these fields.